### DISCUSSION TITLE
Add private notes to project boards

### DISCUSSION THREAD

**Discussion by etsyachristensen** [community/community#16924] [Enterprise]
For enterprise users or even just organizations using github, it would be great to be able to comment on a github issue such that only users in the organization can see the comment, NOT the original poster.

Use case: We have a public Github repo that's used just for issue tracking. Outside developers post issues there for us to review and triage. We have some project boards setup within github.com as part of our organization. When we get an issue posted publicly that needs to be handled, we can add it to the project which is not visible to the outside developer. But we can't make any comments on that issue for internal discussions without the original poster seeing those comments. We are using Github's projects in place of Jira in order to try to streamline some of our processes but it's severely lacking in this area. The only alternative is to create a new issue within an internally used repo to add to the project and reference the original issue. But this is so very redundant and not very intuitive.

**Comment by jaclynjessup** [comment]
Even when you lock the conversation on an issue to be only the contributors any comments are still visible to the public. Having a private section on the issue or within the project would be a massive win!

**Comment by sovetski** [comment]
I confirm that it will be a great feature!!
